### ISSUE TITLE
contact import

### ISSUE THREAD

**Issue by awdrfunk** [nextcloud/server#1168]
importet contacts with "umalute" äöü dosnt supportet, they are wrong
(imoprt from thunderbird adress-book)
here a exemple for "Schärer", it called "Sch=C3=A4rer"

nexcloud version 10
mysql
php 7.0.9

**Comment by nickvergessen** [comment]
How did you import the contacts?

**Comment by awdrfunk** [comment]
with the web-frontend from nc.
is there another way to import?

**Comment by enaut** [comment]
I can confirm that.

1.  can be reproduced by creating a contact in Android contacts
2. export it (to vcf)
3. import it to NC/contacts

The imported file that I used is directly exported by android see the attachment:
[inAndOut.zip](https://github.com/nextcloud/server/files/467557/inAndOut.zip)

I also added the file output.vcf that is created by:

1. import given input.vcf to nc 
2. press the download button in nc

The issue was once reported here: #541
and also here: owncloud/contacts#399
